docs/reference/common-workflows/from-kobotoolbox-to-superset/index.md

```diff
@@ -0,0 +1,211 @@
+
+:::
+   
+
+2. **Set Windmill variables for KoboToolbox**
+
+   The Windmill tool will be used later to fetch the data from Kobotoolbox and into your Data Warehouse.
+
+   You will need to send the following information to your `Windmill Admin`:
+   - Your KoboToolbox server: Either `kc.kobotoolbox.org` or `kf.kobotoolbox.org` .
+   - Your user's API Key: You can get to it in the `Account Settings`, `Security` tab, `API Key`. You show it by clicking on the `DISPLAY` button to the right of that label.
+
+:::info
+
+If you are the `Windmill Admin`, you will have to receive this data and set the account up in Windmill.
+
+This is done by creating a `variable` within Windmill that describes how to access to your KoboToolbox account.
+
+In order to set it up, you need to:
+1. access your Windmill instance, as described below.
+2. Go to the `Variables` section in the left tabs menu.
+3. Create a new variable with the `+ New variable` button in the top right corner of the page and add the following information in the variable form:
+   
+   | Parameter | Value |
+   |------------|-------------|
+   | **Path** | `f/connectors/kobotoolbox` |
+   | **Variable value** | The obtained API Key |
+   | **Description** | Optional detailed explanation |
+
+4. Go to the `Resources` section in the left tabs menu.
+5. Create a new resource with the `Add resource` button in the top right corner of the page and add the following information in the resource form:
+
+   | Parameter | Value |
+   |------------|-------------|
+   | **Resource type**| 
+   | **Path** | `f/connectors/kobotoolbox` |
+   | **Description** | Optional detailed explanation |
+   | **server_url** | `https://kf.kobotoolbox.org` or `https://kc.kobotoolbox.org` |
+   | **api_key** | `$var:f/connectors/kobotoolbox` |
+
+:::
+
+### Data Collection workflow
+
+1. **Create a KoboToolbox project**  
+   
+   Follow the instructions to [create a project](https://support.kobotoolbox.org/quick_start.html#creating-your-first-project).
+
+2. **Add the replication user**  
+
+   In case the owner of the project is not the same as the user used for Guardian Connector defined in the setup, Add it to the KoboToolbox project, here is the [documentation to sharing projects](https://support.kobotoolbox.org/managing_permissions.html).
+   Make sure you provide it with `Manage project` permissions in order for it to be able to pull the data.
+   This account allows Guardian Connector to replicate your data.
+
+3. **Gather project details**  
+
+   You’ll need the following information in the next step:
+   - **KoboToolbox instance**: either `kf.kobotoolbox.org` or `kc.kobotoolbox.org`
+   - **Project ID**: You can find the Project ID in the URL of your KoboToolbox project's Summary page.
+   If the URL looks like this: https://kf.kobotoolbox.org/#/forms/a26w95BYpDoxBJbxDAaSbH/summary
+   The Project ID would be in between "forms/" and "/summary". In this case it would be: a26w95BYpDoxBJbxDAaSbH
```

Comment:
```suggestion
   For example, for a URL like `https://kf.kobotoolbox.org/#/forms/a26w95BYpDoxBJbxDAaSbH/summary`, the Project ID is found in between "forms/" and "/summary". In this case it would be: `a26w95BYpDoxBJbxDAaSbH`
```